src/tools/kata-ctl/Cargo.toml

```
@@ -20,6 +20,8 @@ thiserror = "1.0.35"
 privdrop = "0.5.2"
 nix = "0.25.0"
 
+runtimes = {path="../../runtime-rs/crates/runtimes"}
```

Nit: spacing for consistency with existing format and other manifests:

```suggestion
runtimes = { path = "../../runtime-rs/crates/runtimes"}
```